[5.11.0] Add missing info on allowed-auth-handlers (#3545)
wso2/docs-is: en/docs/develop/authenticating-and-authorizing-rest-apis.md

| **http_method**      | This defines the method as `                 all                `, `                 post                `, `                 get                `, etc. | `                 all                `                     |
| **permissions**      | This defines the user role permission that is required to authorize the resource. You can enter multiple permission strings in a comma-separated list.      | `                 /permission/admin/login                ` |
| **scope**      | This defines scopes that an access token requires to access the resource. You can enter multiple scopes in a comma-separated list.     | `                 internal_idp_create                ` |

> | **allowed_auth_handlers** | This defines the handlers that need to be engaged for the particular resource. If this property is not configured it will be set to the default value `all` which implies all available handlers are engaged to the resource.     | `                 BasicAuthentication                ` |

Reviewer: ```suggestion
| **allowed_auth_handlers** | This parameter defines the handlers that need to be engaged for the particular resource. The default value `all` implies that all available handlers are engaged with the resource.     | `                 BasicAuthentication                ` |
```
